addons/web/static/src/webclient/burger_menu/burger_menu.js

```diff
@@ -63,6 +63,7 @@ export class BurgerMenu extends Component {
     }
     _onMenuClicked(menu) {
         this.menuRepo.selectMenu(menu);
+        this._closeBurger();
```

Comment:
With this, the ACTION_MANAGER:UPDATE event listener is, *I believe*, no longer required.
You may remove it, see if no tests break and just check if everything seems alright. 
If at worst bugs are discovered later because of the change (like idk burger menu is not closed with a new action because you quickly closed it and reopened it during loading) it will be a good opportunity to fix and add a test.